app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -159,6 +163,42 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
         components.appStartupTelemetry.onFenixApplicationOnCreate()
     }
 
+    private fun restoreBrowsingSession() = GlobalScope.launch(Dispatchers.Main) {
```

Comment:
nit: Naming suggestion in A-C PR, but let's keep consistent based on our decision.